src/components/DropDownMenu.tsx

```diff
@@ -13,7 +13,7 @@ interface DropDownMenuProps {
 }
 
 /** A custom drop down menu. */
-// eslint-disable-next-line react/display-name
```

Comment:
?

Reply:
Basically as per eslint v9.

if there is no rule define in eslint.config.ts and we are adding the command in our code. so eslint lint will show that command as warning.

so we just fix all these by running yarn lint --fix. if you want to keep these warning as it is so will revert all these.